### PR TITLE
Fix missing database connection file

### PR DESCRIPTION
**Pull request by khyechun**
## Context

JIRA ticket: None

## Checklist

- [x] My branch is up-to-date with the `main` branch
- [ ] Unit test written
- [ ] Acceptance Criteria considered and implemented

## Risks
Low

### CODE REVIEW

**Review by lanceljr — APPROVED** [review]
Solid work kc!

**Review by lanceljr — CHANGES_REQUESTED** [review]
Dear Khye Chun, 

it seems like the database.js file is still not included.

**Comment by Randyyeo** [comment]
About this, i added the database file to `.gitignore` as for mac my configurations are different. Can we instead update the readme file to include instructions to create the `database.js` file? WE can have a template there

File: backend/database/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var mysql = require("mysql");
```
Comment (Randyyeo) [inline backend/database/index.js:1]: Can we delete this file? I added the reason in the comments above

**Review by Randyyeo — APPROVED** [review]

**Review by Randyyeo — APPROVED** [review]
Thanks sir @khyechun

**Review by lanceljr — APPROVED** [review]

**Pull request merged**